docs/StartANewReactApp.md

```diff
@@ -0,0 +1,9 @@
+# How to Start a New ReactJS Project
+
+
+1. Create a directory for your project inside of `vets-website/src/js/`. 
+2. Create an entry file in your project directory using the project name to prefix the filename. Example: `vets-website/src/js/[project-name]/[project-name]-entry.jsx`.
```

Comment:
If you're using education benefits for all the examples here, I might move the line you have above with `[project-name]` in it before the example and have the example actually be the `edu-benefits` entry file name. So:
```
Create an entry file for your project directory using the project name to prefix the filename: `vets-website/src/js/[project-name]/[project-name]-entry.jsx`. Example: `vets-website/src/js/edu-benefits/edu-benefits-entry.js`.
```